### ISSUE TITLE
DPI Issue

### ISSUE THREAD

**Issue by itsaaash** [LNReader/lnreader#221]
![Screenshot_20220402-043610_LNReader](https://user-images.githubusercontent.com/91327924/161869075-6b8c6de0-0173-4c26-9cfd-d6cf5d508aa3.png)
Half the texts are hidden, doesn't get fixed until i lower or change the DPI. I'm on A12, the issue starts happening when I'm over 400+ DPI usually.

**Comment by nyagami** [comment]
![image](https://user-images.githubusercontent.com/86464880/230857155-5a033bd1-2298-40d7-b4a3-631760021328.png)
![image](https://user-images.githubusercontent.com/86464880/230857198-472f3f98-a490-4090-b01a-93df4d2a75e3.png)
Yes, I figured out why this bug happend. 
If title is too long (2+ lines), and the <Text> component will trunct the string (dont know why it trunct so damn).
Should remove numberOfLines and use string.slice instead. 🤔 maybe 45 for max length

**Comment by rajarsheechatterjee** [comment]
> Should remove numberOfLines and use string.slice instead. 🤔 maybe 45 for max length

I don't think slicing it would be the correct solutions as the text should take the available space in different resolutions.

**Comment by nyagami** [comment]
Yeah, but I think it's fine for almost cases. I chose 45 because on my emulator, 40 and 50 are limits which overflow string is still on one line with

**Comment by rajarsheechatterjee** [comment]
> Yeah, but I think it's fine for almost cases. I chose 45 because on my emulator, 40 and 50 are limits which overflow string is still on one line with

Not for tablets or horizontal orientation.
